### DISCUSSION TITLE
NCP Support

### DISCUSSION THREAD

**Discussion by theHurc** [openthread/openthread#7938] [Q&A] [answered]
I had previously been using the NCP with the `wpantund` service successfully. I decided to upgrade OT only to find that `wpantund` isn't supported any longer in favor of `ot-br-posix` in an RCP model. The NCP model works so much better in my application because my radio chip (the NCP) is very powerful and my border router has many other applications which need all the resources they can get.

I have a few questions:

Can I still use the NCP model with the latest `ot-br-posix`? or has that totally gone away?

`https://github.com/openthread/borderrouter` now redirects to `https://github.com/openthread/ot-br-posix` but it seems to have changed significantly. Is that a standard way to do things on github? Seems like a a confusing way to manage a repo to me.

Where is the last commit the NCP model was supported so I can still use it?

Any other comments or guidance would be helpful.

**Comment by jwhui** [comment, marked as answer]
The latest ot-br-posix main branch only supports the RCP model.

The latest commit supporting NCP model is maintained in a ot-br-posix branch: https://github.com/openthread/ot-br-posix/tree/wpantund/master Unfortunately, it hasn't been updated for quite sometime.
